### ISSUE TITLE
Is it possible to map esm3 embedding back to sequence?

### ISSUE THREAD

**Issue by johnnytam100** [evolutionaryscale/esm#94]
I want to explore the esm3 space but wondering how to map the modified embedding back to sequence.

**Comment by prwoolley** [comment]
Not a developer, but it would be easier to map the modified embedding through the language modeling head to get the sequence logits (P(sequence|embeddings)). Mapping backwards (creating the inverse function for ESM3) might not be reliable or deterministic since multiple inputs can embed similarly.

**Comment by ebetica** [comment]
@johnnytam100 you would probably have to train your own model to do this, ESM3 does not natively learn any nice manifold in embedding space.

**Comment by johnnytam100** [comment]
Just curious to know: how a nice manifold vs not nice manifold would affect the outcome of this embed -> seq task?
